lambda/cfnresponse.py

```diff
@@ -15,30 +15,27 @@
 def send(event, context, responseStatus, responseData, physicalResourceId=None):
     responseUrl = event['ResponseURL']
 
-    print responseUrl
+    print(responseUrl)
 
-    responseBody = {}
-    responseBody['Status'] = responseStatus
-    responseBody['Reason'] = 'See the details in CloudWatch Log Stream: ' + context.log_stream_name
-    responseBody['PhysicalResourceId'] = physicalResourceId or context.log_stream_name
-    responseBody['StackId'] = event['StackId']
-    responseBody['RequestId'] = event['RequestId']
-    responseBody['LogicalResourceId'] = event['LogicalResourceId']
-    responseBody['Data'] = responseData
+    response_body = {'Status': responseStatus,
+                     'Reason': 'See the details in CloudWatch Log Stream: ' + context.log_stream_name,
+                     'PhysicalResourceId': physicalResourceId or context.log_stream_name, 'StackId': event['StackId'],
```

Comment:
```suggestion
                     'PhysicalResourceId': physicalResourceId or context.log_stream_name, 
                     'StackId': event['StackId'],
```
For better readability, I would suggest to have one attribute per line.